test_scripts/breakdancer_test.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
May want to add `-ex` or smth to get early errors and descriptions of the commands output.  Not sure if this comes for free in Travis tho